frontend/packages/console-app/src/components/pdb/form-popover.scss

```diff
@@ -0,0 +1,16 @@
+.pdb-form-popover {
+    &__title {
+      margin-top: var(--pf-global--spacer--md);
+      margin-bottom: var(--pf-global--spacer--sm);
+    }
+    &__description {
+      font-size: var(--pf-global--FontSize--sm);
+    }
+  }
```

Comment:
I don't think we need these. `Stack` (with `hasGutter`) and `StackItem` cover this.

Also, not sure what the plan here was for the font-size of the descriptions, but that's not needed I don't think 🤔 

Recommending deleting and seeing my next comment for the React elements.